client/lib/analytics/index.js

```diff
@@ -344,6 +354,31 @@ const analytics = {
 		}
 	},
 
+	// Lucky Orange tracking
+	luckyOrange: {
+		initialized: false,
```

Comment:
Since this bould down to state management, it might make sense to bring that into Redux. To that end, you need to add a `reducer.js` to `state/analytics` that includes a `continuousTracking` reducer that reflects this, and is updated by the `ANALYTICS_CONTINOUS_MONITOR` event. Plus you'll also need a selector...